js/index.js

```diff
@@ -16,7 +16,7 @@ app.controller("IndexController", ["$scope", "$http", function($scope, $http) {
   })
 
   $scope.markers = new Array();
-
+  $scope.counter = 0;
```

Comment:
Is it not possible to define this dynamically, something like `$scope.counter = $scope.markers.length`? I'm worried that these could get out of sync, and when we load them from the mongo database, the count won't be initialized properly.